### PR TITLE
Feat: Add Ingredient class and add syntax changes to RecipeRepository

### PR DESCRIPTION
**Pull request by tfettig22**
- What's this PR do: Add Ingredient class and test file.  Also minor changes to syntax of RecipeRepository and refactored how the data was being imported from the test files.
- Where to start review: Please look over Ingredient class and test file to make sure it all makes sense, I did confirm that all tests are passing and the class should be dynamic. 
- How to test: If you want to verify, you should be able to pull the branch down and run the tests in your console
- Background context: I learned a better way to import the data files without nesting them too deep in another object, let me know if you have questions but overall I think this is easier to read and makes more sense.

### CODE REVIEW

File: src/classes/Ingredient.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class Ingredient {
```
Comment (GOECHA) [inline src/classes/Ingredient.js:1]: Reviewed, all tests pass. Thank you.

File: test/Ingredient-test.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
+import { expect } from 'chai';
```
Comment (GOECHA) [inline test/Ingredient-test.js:1]: Reviewed, all tests pass. Thank you.

File: src/classes/RecipeRepository.js [hunk 3]
```diff
@@ -1,10 +1,6 @@
-const data = require('../data/recipes')
-
 class RecipeRepository {
   constructor(recipeData) {
     this.recipeData = recipeData
-
-    // One class to get you started!
   }
 
   filterByTag(tag) {
```
Comment (GOECHA) [inline src/classes/RecipeRepository.js:6]: Reviewed, TY.

File: test/RecipeRepository-test.js [hunk 4]
```diff
@@ -11,26 +11,29 @@ describe('Recipe', () => {
     repository = new RecipeRepository(testData);
   });
   
-  it('Should be a function', () => {
+  it('should be a function', () => {
     expect(RecipeRepository).to.be.a('function');
   });
 
-  it('should be able to take in recipe data as an argument', () => {
+  it('should be an instance of RecipeRepository', () => {
```
Comment (GOECHA) [inline test/RecipeRepository-test.js:18]: Great addition to testing. It is more thorough.

**Review by GOECHA — APPROVED** [review]
Changes, approved. Thank you.

**Review by forsethnico — COMMENTED** [review]
Looks great to me! I liked that you added multiple tests cases for each ingredient test

**Review by forsethnico — APPROVED** [review]

**Pull request merged**